app/models/service_template_load_balancer.rb

```diff
@@ -0,0 +1,23 @@
+class ServiceTemplateLoadBalancer < ServiceTemplate
+  include ServiceLoadBalancerMixin
+
+  before_save :remove_invalid_resource
+
+  def remove_invalid_resource
+    # remove the resource from both memory and table
+    service_resources.to_a.delete_if { |r| r.destroy unless r.resource }
+  end
+
+  def create_subtasks(_parent_service_task, _parent_service)
+    # no sub task is needed for this service
+    []
+  end
+
+  def self.default_provisioning_entry_point
```

Comment:
@bzwei hm, that one is still a wip, so this would be probably also followup PR once that PR is merged? Otherwise this code won't work now.